projects/bootloader/src/config.c

```diff
@@ -11,41 +11,31 @@ static BootloaderConfig s_config_1_blob = { 0 };
 static PersistStorage s_config_2_persist = { 0 };
 static BootloaderConfig s_config_2_blob = { 0 };
```

Comment:
Initialize the `BootloaderConfig` blobs to sensible defaults of the parameters - I'd say these should be the values of the fields:
- `controller_board_id` should be `63` so that it's easy for us to tell that it's invalid, since 63 is the maximum controller board ID
- `controller_board_name` should be the empty string (or maybe `"<unset>"`)
- `project_present` should be false
- `project_name`, `project_info`, and `git_version` should be the empty string
- `application_crc32` and `application_size` should be zero

Most of these are the same as the defaults, but it's good to specify them explicitly, and we don't want to use the default controller board ID of 0 since that might cause errors (ID 0 is reserved for the client script). It might in fact be a good idea to factor out the default controller board ID into a `#define` with a comment so people know why 63 was chosen.

The `crc32` field is what I was talking about with how we should dynamically calculate it. At the very start of `config_init`, just call `prv_compute_crc32` on both of the config blobs and set their crc32 fields to the result. That will be more robust than handling the default config after having loaded the config - it's best to have a correct config regardless and then we don't need special cases.